packages/components/report.20200330.205136.6288.0.001.json

```diff
@@ -0,0 +1,553 @@
+
```

Comment:
I think we should exclude the reports from git, unless there is a reason to keep them?

Reply:
will remove